src/components/menu/menu.jsx

```diff
@@ -32,22 +33,40 @@ MenuComponent.propTypes = {
     place: PropTypes.oneOf(['left', 'right'])
 };
 
-const MenuItem = ({
-    children,
-    className,
-    onClick
-}) => (
-    <li
-        className={classNames(styles.menuItem, className)}
-        onClick={onClick}
-    >
-        {children}
-    </li>
-);
+// you can pass MenuItem an onClick function. If none is found, when clicked it
+// looks for an href and navigates the web page to that.
+class MenuItem extends React.Component {
```

Comment:
Things in `src/components` ideally should just be simple render functions, and not have any behavior. If you need to add behavior, use a container that passes a prop to this component.  The `MenuBar` component violates this convention, but we should try to keep that at bay.